homeassistant/components/bmw_connected_drive/binary_sensor.py

```diff
@@ -107,7 +109,11 @@ def is_on(self):
     def device_state_attributes(self):
         """Return the state attributes of the binary sensor."""
         vehicle_state = self._vehicle.state
-        result = {"car": self._vehicle.name}
+        result = {
+            "car": self._vehicle.name,
+            "update_time": vehicle_state.timestamp,
```

Comment:
Why do we need this? This is already the same as `last_updated` of the state?

Reply:
This always shows the correct time the car has made the last update, also after HA is restarted. That's the reason I use this attribute in notifications.

{{ states.binary_sensor.x1_sdrive18i_windows.last_updated }} -> 2020-02-24 21:57:36.896197+00:00

{{ states.binary_sensor.x1_sdrive18i_windows.attributes.update_time }} -> 2020-02-24 17:09:22+00:00

Reply:
This is a workaround for a flaw in Home Assistant.

Reply:
My suggestion is to keep this in as this attribute always contains the right time and gives an easy overview of when the car has sent the last update.

![bimmer](https://user-images.githubusercontent.com/11230573/75199266-f3633c80-5762-11ea-99b5-34a2557d52b1.jpg)

Reply:
What to do with `update_time`?

Reply:
I removed `update_time`.